### PR TITLE
Update Winget argument description for commands

### PR DESCRIPTION
**Pull request by ryfu-msft**
Changes:
- Added --architecture argument description for Install Command
- Added --dependency-source argument description for Install Command
- Removed --location argument description for List Command
- Added --include-unknown argument description for Upgrade Command

### CODE REVIEW

**Comment by PRMerger8** [comment]
@ryfu-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@ryfu-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by GitHubber17** [comment]
@KevinLaMS @ryfu-msft - After the changes are approved, the review team will move the commits to the private repo. Thank you

File: hub/package-manager/winget/export.md
Comment (mattwojo) [inline hub/package-manager/winget/export.md:40]: Would this also work with -v?

File: hub/package-manager/winget/hash.md
Comment (mattwojo) [inline hub/package-manager/winget/hash.md:27]: Same ? as above... would -v work?

File: hub/package-manager/winget/import.md
Comment (mattwojo) [inline hub/package-manager/winget/import.md:39]: -v?

File: hub/package-manager/winget/install.md
Comment (mattwojo) [inline hub/package-manager/winget/install.md:46]: Should we provide an example?
"Select the architecture to install. For example, --architecture x86."

File: hub/package-manager/winget/install.md
Comment (mattwojo) [inline hub/package-manager/winget/install.md]: Should this include an example? 
--dependency-source https://www.contoso.com/cache

(not sure if this is correct usage)

File: hub/package-manager/winget/import.md
Comment (ryfu-msft) [reply hub/package-manager/winget/import.md:39]: I don't think --verbose-logs has a short form since that would conflict with other commands that use -v like winget install which uses -v to specify --version.

Comment (mattwojo) [reply hub/package-manager/winget/import.md:39]: Ok, sounds good. Thanks for confirming.

File: hub/package-manager/winget/install.md
Comment (ryfu-msft) [reply hub/package-manager/winget/install.md:46]: Will defer to @denelon as he may have a stronger opinion. 

The changes I made are exactly aligned with what I see in the client help display.  I am hesitant to add an example here as that would make it seem slightly out of place. Is there a better place we can show examples?

File: hub/package-manager/winget/install.md
Comment (denelon) [reply hub/package-manager/winget/install.md]: We haven't implemented any support for --dependency-source, so I agree we should either remove it, or explicitly call out that it is not yet supported in the client.

File: hub/package-manager/winget/install.md
Comment (denelon) [reply hub/package-manager/winget/install.md:46]: I think we have some examples, and we should include one for arm64 as an example.

**Comment by PRMerger12** [comment]
@ryfu-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/package-manager/winget/install.md
Comment (ryfu-msft) [reply hub/package-manager/winget/install.md]: okay, I did not know that we don't have support for that yet. I removed it for now and will add it back when it is supported

File: hub/package-manager/winget/install.md
Comment (ryfu-msft) [reply hub/package-manager/winget/install.md:46]: I decided to not give an example here so that it is consistent with the other argument descriptions that also don't have examples. I think just a clear description here would be sufficient.

**Comment by mattwojo** [comment]
@GitHubber17 - Thank you for the offer to move these commits to the private repo. Do you know whether this is necessary? I believe this repo is set up to mirror between our public and private repos, so the changes would be automatically reflected on the private once this is merged. ..but I'll default to your judgement.

**Comment by mattwojo** [comment]
#sign-off

**Comment by PRMerger7** [comment]
@mattwojo: I'm sorry - only the author of this article, @KevinLaMS, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by denelon — APPROVED** [review]

**Review by KevinLaMS — APPROVED** [review]

**Pull request merged**
